README.md

```diff
@@ -0,0 +1,2 @@
+# surveyform
+Survey form project
```

Comment:
It is best practice to have a fully documented and descriptive README.md file, as it would allow other possible teammates to understand what the repository is for and how to install everything that is necessary to work with/on it. I recommend using this template: https://github.com/microverseinc/readme-template.